bluejeans/README.md

```diff
@@ -16,10 +16,11 @@ After you've the credentials, we need two secrets named `bluejeansAppKey` & `blu
 
```

Comment:
> `bluejeansAppSecret` with your credentials as their values. You can create them by running `/nc secret_create`.`

Remove grave accent (`) at the end as well.

Reply:
Done.